views/views.py

```diff
@@ -58,6 +59,7 @@ def get_challenge(level: str, name: str):
         challenges_groupby_level=challenge_manager.challenges_groupby_level,
         code_under_test=challenge.user_code,
         test_code=challenge.test_code,
+        hints_for_display=render_hints(challenge.hints) if challenge.hints else None,
```

Comment:
I think it's perfectly fine to just use `hint` instead of `hints_for_display`

Reply:
I use a long name to indicate that it contains a "rendered" hints content, would prefer to leave it as is.